media/css/m24/pencil-banner.scss

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+@import 'vars/color';
+@import 'vars/fonts';
+@import 'vars/grid';
+@import 'vars/spacing';
+@import 'vars/text';
+
+.m24-pencil-banner {
```

Comment:
**Issue:** on mobile the pencil banner overlaps the mobile navigation menu, making it impossible to close

<img width="414" alt="image" src="https://github.com/user-attachments/assets/ad966126-66a0-4af7-96d9-47111577c313">